src/frontend/packages/lib_common/src/queries/deleteOne.ts

```diff
@@ -21,4 +23,6 @@ export const deleteOne: MutationFunction<
 
     throw { code: 'exception' };
   }
+
+  return undefined as unknown as Promise<T>;
```

Comment:
```suggestion
  return undefined as unknown as T;
```

Reply:
in fact, with https://github.com/openfun/marsha/pull/1744/files#r987706362 : 
```suggestion
  return undefined;
```

Reply:
I agree it is not elegant but it respect the type `MutationFunction`.
`return undefined;` will not work, you must return a `Promise<T>` or `Promise<any>` otherwise you will get an error here: `\src\frontend\apps\lti_site\data\queries\index.tsx`.
`Promise<any>`  is not **EsLint** friendly, so you have  `Promise<T>` left.
I tried to cast `response.json()`, but the test gave me some errors if I well remember, so an easy solution is 
`return undefined as unknown as Promise<T>;` or `return void as unknown as Promise<T>;`, that stick to the legacy.

I am open to suggestion.

Reply:
It is working [76b86c2](https://github.com/openfun/marsha/pull/1744/commits/76b86c2c379eeed1a30b263a7e7f31d1a3f7d42a)
You are in an async function, the return is always a promise.